lib/cc/engine/bundler_audit/analyzer.rb

```diff
@@ -15,10 +16,14 @@ def initialize(directory:, engine_config_path: DEFAULT_CONFIG_PATH, stdout: STDO
 
         def run
           raise(GemfileLockNotFound, "No Gemfile.lock found.") unless gemfile_lock_exists?
-          return unless gemfile_lock_in_include_paths?
+          unless gemfile_lock_in_include_paths?
+            stderr.puts("Not auditing #{gemfile_lock_relative_path} because it is not in include_paths")
+            return
+          end
 
           Dir.mktmpdir do |dir|
             FileUtils.cp(gemfile_lock_path, File.join(dir, GEMFILE_LOCK))
+            FileUtils.cp(gemfile_path, File.join(dir, GEMFILE))
```

Comment:
Are you sure this is needed?

I ask because I don't think it is, and not doing so would reduce this diff substantially.